web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -421,11 +425,12 @@
 
 
       //Update Links for social media
+      $scope.showSocialBarInFooter = gnGlobalSettings.gnCfg.mods.footer.showSocialBar;
       $scope.socialMediaLink = $location.absUrl();
       $scope.$on('$locationChangeSuccess', function(event) {
         $scope.socialMediaLink = $location.absUrl();
         $scope.showSocialMediaLink =
-            ($scope.socialMediaLink.indexOf('/metadata/') != -1);
+          (($scope.socialMediaLink.indexOf('/metadata/') == -1) && ($scope.showSocialBarInFooter));
```

Comment:
Not a big fan of the original logic here, and your change make a it a bit more obscure.

A cleaner way might be:
```js
$scope.getSocialLinksVisible = function() {
  var onMdView =  $location.absUrl().indexOf('/metadata/') > -1;
  return !onMdView && gnGlobalSettings.gnCfg.mods.footer.showSocialBar,
}
```

Then use `ng-show="getSocialLinksVisible()"` in the template.

Reply:
Yes, that looks better. I was indeed struggling with the original code, where the variable was called `showSocialMediaLink` but an `ng-hide` was used.

I will change it